heron/api/src/scala/com/twitter/heron/streamlet/scala/impl/StreamletImpl.scala

```diff
@@ -13,25 +13,28 @@
 //  limitations under the License.
 package com.twitter.heron.streamlet.scala.impl
 
+import scala.collection.JavaConverters
+
 import com.twitter.heron.streamlet.{
   JoinType,
   KeyValue,
   KeyedWindow,
   SerializableTransformer,
+  Streamlet => JavaStreamlet,
   WindowConfig
 }
+import com.twitter.heron.streamlet.impl.{StreamletImpl => JavaStreamletImpl}
 import com.twitter.heron.streamlet.impl.streamlets.SupplierStreamlet
+
 import com.twitter.heron.streamlet.scala.{Sink, Streamlet}
 import com.twitter.heron.streamlet.scala.converter.ScalaToJavaConverter._
 
 object StreamletImpl {
 
-  def toScalaStreamlet[R](
-      javaStreamlet: com.twitter.heron.streamlet.Streamlet[R]): Streamlet[R] =
+  def toScalaStreamlet[R](javaStreamlet: JavaStreamlet[R]): Streamlet[R] =
```

Comment:
suggest to rename to fromJavaStreamlet(). The current class is streamlet.scala.impl. StreamletImpl, so I feel fromJavaStreamlet() makes more sense.

Reply:
@nwangtw Thanks again for the review. Addressed.